allennlp/tools/drop_eval.py

```diff
@@ -243,5 +254,11 @@ def evaluate_prediction_file(prediction_path: str, gold_path: str) -> Tuple[floa
                         required=False,
                         default="sample_predictions.json",
                         help='location of the prediction file')
+    parser.add_argument("--output_path",
+                        type=str,
+                        required=False,
+                        default=os.devnull,
```

Comment:
I think you should hide this behind an `if` in the main logic, instead of trying to write to devnull.

Reply:
I'm not sure I understand, what's the problem with writing to devnull?
Another option is to add a default output file, would that be better?

Reply:
It's just better to avoid it, I think. I don't know how it interacts with docker or other things. It's an unnecessary complexity that we can avoid just with a default value of `None`.

Reply:
Got it, makes sense.
I changed this to use None instead.